client/my-sites/menus/item-options/test/test-posts.jsx

```diff
@@ -1,60 +1,60 @@
-
-require( 'lib/react-test-env-setup' )();
-
 /**
  * External Dependencies
  */
-var	sinon = require( 'sinon' ),
-	sinonChai = require( 'sinon-chai' ),
-	chai = require( 'chai' );
+import chai from 'chai';
+import path from 'path';
 
 /**
  * Internal Dependencies
  */
-var Posts = require( '../posts' ),
-	siteMenus = require( 'lib/menu-data' ),
-	helpers = require( 'my-sites/pages/helpers' );
-
-chai.use( sinonChai );
-
-describe( 'Posts' , function () {
-	describe( 'maybeInjectPosts', function () {
+import useFakeDom from 'test/helpers/use-fake-dom';
+import { useSandbox } from 'test/helpers/use-sinon';
+import useFilesystemMocks from 'test/helpers/use-filesystem-mocks';
+
+describe( 'Posts', function() {
+	let Posts, siteMenus, helpers;
+	useFakeDom();
+	useSandbox();
+
+	const menuDataPath = path.dirname( require.resolve( 'lib/menu-data' ) );
+	useFilesystemMocks( menuDataPath + '/test' );
```

Comment:
i think this is actually OK. I like that it requires a bit of gymnastics to do something odd.